content/en/testing-guide/_index.adoc

```diff
@@ -0,0 +1,62 @@
+
+---
+title: "PAPRa Testing Guide (13 March 2021 Edition)"
+linkTitle: "PAPRa Testing Guide"
+weight: 1
+description: >
+  How we tested the PAPRa to arrive at claims of N95/N99 levels of protection
+---
+
+== Quick Synopsis
+
+This page will walk you through how we established that the PAPRa can offer N95 levels of protection on a mask when worn by someone with a beard, and N99 levels of protection by the fan box itself.  These tests are by no means exhaustive; far more testing is needed to ensure that these results can be achieved consistently, and the testing should be done in a more controlled environment than what we have done.
+
+Furthermore, we offer no warranty, explicit or implied, that any device that you create will achieve similar standards of filtration.  We are currently working to determine the best path forward for individuals who make their PAPRs to test them.
+
+== Parts Needed
+
```

Comment:
Should there be a tabulated bill of materials (for testing) here?

Reply:
Yes, but I don't have one

Reply:
opened an [issue](https://github.com/tetrabiodistributed/papra/issues/11)